src/parser/warrior/arms/modules/spells/azeritetraits/ExecutionersPrecision.js

```diff
@@ -7,30 +7,36 @@ import SPELLS from 'common/SPELLS';
 import Analyzer from 'parser/core/Analyzer';
 import Enemies from 'parser/shared/modules/Enemies';
 
-import SpellUsable from '../features/SpellUsable';
+import SpellUsable from '../../features/SpellUsable';
 
-class ExecutionAnalyzer extends Analyzer {
+/**
+ * Execute increases the damage of your next Mortal Strike against the target by 439, stacking up to 2 times.
+ */
+
+class ExecutionersPrecisionAnalyzer extends Analyzer {
   static dependencies = {
     enemies: Enemies,
     spellUsable: SpellUsable,
   };
 
-  executes = 0;
-  wastedExecutionersPrecisions = 0;
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTrait(SPELLS.EXECUTIONERS_PRECISION_TRAIT.id);
+  }
+
+  procs = 0;
+  wastedProcs = 0;
 
   on_byPlayer_cast(event) {
-    if (SPELLS.EXECUTE.id !== event.ability.guid) {
-      return;
-    }
+    if (SPELLS.EXECUTE.id !== event.ability.guid) return;
 
```

Comment:
Same as earlier, we prefer using the approach that was there before for consistency.